cfpb/cfgov-refresh
# Add sidefoot_heading field to ReusableText snippets

The current Wagtail rich text editor does not allow for the styling of a heading with our green bar "slug" styling that we use to denote sections in the sidebar or pre-footer. This PR adds a new field to ReusableTexts that will create a slug-style heading if it is given text.

See some discussion beginning here: https://[GHE]/design-ops/CEE-projects/issues/189#issuecomment-116270 (warning: long thread, will take a while to load)

## Additions

- `sidefoot_heading` field for `ReusableText` snippets

## Changes

- Renames migration 0067 to have a useful name and updates migrations that depended on it accordingly.

## Testing

1. Pull branch
2. Migrate
3. Run server
4. Edit a reusable text snippet to have something in its `sidefoot_heading` field
5. Preview a page with that reusable text snippet inserted and see the rendered heading

## Screenshots

Interface:

<img width="875" alt="screen shot 2017-06-19 at 12 06 30" src="https://user-images.githubusercontent.com/1044670/27394933-086d1866-567c-11e7-96d8-f0fa7b12e814.png">

Result:

![screen shot 2017-05-25 at 17 13 14](https://user-images.githubusercontent.com/1044670/27394950-1a58734a-567c-11e7-8b1f-45692fce9b73.png)

## Checklist
